habushu-mixology-consumer/pyproject.toml

```diff
@@ -11,6 +11,10 @@ python = "^3.11"
 [tool.poetry.group.monorepo.dependencies]
 habushu-mixology = {path = "../habushu-mixology", develop = true}
 
+
+[tool.poetry.group.dev.dependencies]
+behave-cucumber-formatter = "^1.0.1"
```

Comment:
S: generally, we are trying to go with `>=` over `^` given that it plays more nicely general Python upgrade approaches (if there is a good reason to use `^` instead, that's fine - but recommend throwing a comment next to it)

Reply:
You're right -- doing ^ would not fetch major and minor releases e.g. 2.0.0 or 1.1.0